### DISCUSSION TITLE
Nerd Fonts

### DISCUSSION THREAD

**Discussion by XXiaoA** [wezterm/wezterm#2912] [Q&A]
Nerd fonts are not in a same size, it's s misalignment? I tried using font patcher in https://www.nerdfonts.com/ but it didn't work

![Screenshot_20230102_161858](https://user-images.githubusercontent.com/62557596/210211281-eed38c52-a417-4680-9dcb-2c7cb379ab7d.png)

Then I use `JetBrainsMono Nerd Font`, everything is okay
![Screenshot_20230102_161214](https://user-images.githubusercontent.com/62557596/210211417-46f7d09c-6c85-44a0-bc51-0731b4e9b71e.png)

**Comment by faradnikman** [comment]
hi @XXiaoA, correct me if I am wrong, you're using Nerd Font mono such as this [JetBrains Mono Regular Nerd Font Complete Mono.ttf](https://github.com/ryanoasis/nerd-fonts/blob/master/patched-fonts/JetBrainsMono/Ligatures/Regular/complete/JetBrains%20Mono%20Regular%20Nerd%20Font%20Complete%20Mono.ttf) in your first picture. The second one is [JetBrains Mono Regular Nerd Font Complete ~~Mono~~.ttf](https://github.com/ryanoasis/nerd-fonts/blob/master/patched-fonts/JetBrainsMono/Ligatures/Regular/complete/JetBrains%20Mono%20Regular%20Nerd%20Font%20Complete.ttf) (without the `Mono` at the end). Wezterm now bundles in support for _Symbols Nerd Font_ (see #1521 and [here](https://wezfurlong.org/wezterm/changelog.html?highlight=nerd%20font#20220319-142410-0fcdea07)). With this release, you can use a non-patched font as an alternative.

try this command for troubleshooting fonts ([guides](https://wezfurlong.org/wezterm/config/fonts.html?highlight=ls-font#troubleshooting-fonts))

```bash
wezterm ls-fonts | grep "Symbols Nerd Font"
```

hope this helps

**Reply by wez** [reply]
That's correct. You may also want to look at:
* https://github.com/wez/wezterm/discussions/2830

for information on how to fine tune the size to your taste

**Reply by XXiaoA** [reply]
Ummm, no. In the first picture, I use a https://github.com/subframe7536/Maple-font/
Look at the video. If I use any font without builtin nerd font, the nerd fonts are wierd in wezterm. But in other terminal, it look well.

https://user-images.githubusercontent.com/62557596/211241614-2e6169e0-b56d-43cd-9840-c4b47829f1a3.mp4

**Reply by faradnikman** [reply]
Hi @XXiaoA, I can pinpoint your issue, instead of using _Maple Mono SC NF_, you can use _Maple Mono_ instead, the condensed icons comes from the patched font. My take is that the patched font is using `-s` flag which produce **single glyph icon** using [Font Patcher](https://github.com/ryanoasis/nerd-fonts#font-patcher)

> all the asset here is taken from [Maple Mono Font v6.0](https://github.com/subframe7536/Maple-font/releases/tag/v6.0)

## with [Maple Mono SC NF](https://github.com/subframe7536/Maple-font/releases/download/v6.0/MapleMono-SC-NF.zip)
![Screenshot 2023-01-09 at 1 05 12 PM](https://user-images.githubusercontent.com/19277970/211243961-6e0a80b0-f757-4b2f-9e4e-9de473250273.png)

## with [Maple Mono](https://github.com/subframe7536/Maple-font/releases/download/v6.0/MapleMono.zip) ( utilising Symbol Nerd Font fallback )
![Screenshot 2023-01-09 at 1 04 10 PM](https://user-images.githubusercontent.com/19277970/211243883-ab88b5eb-8358-4987-a6ec-d447d119d523.png)

hope this helps

**Reply by XXiaoA** [reply]
Oh, really thanks. This issues have confused me for along time
